src/components/TextArea/TextArea.tsx

```diff
@@ -1,45 +1,18 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import '../../yamui';
 import * as React from 'react';
-import { getBaseTextFieldProps, BaseTextFieldProps } from '../BaseTextField';
-import DebouncedOnChange, { DebouncedOnChangeProps, DebouncedOnChangePrivateProps } from '../../util/DebouncedOnChange';
-import { TextField as FabricTextField } from 'office-ui-fabric-react/lib/TextField';
 import { join } from '../../util/classNames';
-import '../BaseTextField/BaseTextField.css';
-import './TextArea.css';
+import { Omit } from '../../util/types';
+import BaseTextField, { TextAreaProps as InternalTextAreaProps } from '../internal/TextField';
 
-export interface TextAreaProps extends BaseTextFieldProps, DebouncedOnChangeProps {
-  /**
-   * Height of the field, in number of rows.
-   */
-  rows?: number;
-
-  /**
-   * Whether or not to auto adjust textField height.
-   */
-  autoAdjustHeight?: boolean;
-}
+export type TextAreaProps = Omit<InternalTextAreaProps, 'multiline'>;
```

Comment:
i've used `Omit` to remove the `multiline` prop from the public TextFieldProps and TextAreaProps interfaces while the TextField and TextArea component manually set it to true/false. this way the underlying `internal/TextField` component has type safety around multiline (TextArea) vs. single line (TextInput), and the consumer app/engineer can just use the public TextArea or TextInput component without knowing the implementation